code/modules/materials/material_sheets.dm

```diff
@@ -247,6 +247,21 @@
 	default_type = DEFAULT_WALL_MATERIAL
 	icon_has_variants = TRUE
 
+/obj/item/stack/material/steel/attackby(obj/item/W, mob/user)
+	. = ..()
+	if(is_sharp(W))
```

Comment:
Maybe check damage here too, because cutting steel wool out of steel sheets with a piece of shrapnel is, well... a bit dumb.

Reply:
Knives only have 6 force when made out of steel, which is also pretty weird.
![image](https://user-images.githubusercontent.com/22774890/107879707-1db58900-6ee3-11eb-8f0b-9707730e0910.png)